src/components/AddMemorialForm.vue

```diff
@@ -81,23 +81,22 @@
                     </b-form-group>
                 </b-col>
               </b-row>
-              <b-row class="justify-content-center">
+               <b-row class="justify-content-center">
                <b-col cols="8" sm="8">
                   <b-form-group
-                    :label="$t('addMemorialForm.location')"
+                    :label="$t('addMemorialForm.location1')"
                   >
                     <b-form-input
                       list="my-list-id-2"
-                      v-model="memorial.location"
+                      v-model="memorial.location1"
```

Comment:
please don't use variable names like location1, that kind of suffix is not useful - prefer a name which describes whty this location object is different